### ISSUE TITLE
Include blender executable

### ISSUE THREAD

**Issue by lw64** [flathub/org.godotengine.Godot#125]
Currently it is inconvenient to setup and not secure to use blender installed on the host. Also on image based operating systems you cannot modify the host and install for example blender. And using blender installed through flatpak is even more inconvenient to set up.
Because of this I would really like it if a blender executable is included by default in the godot flatpak, so that zero setup time is needed and it just works out of the box.

**Comment by Calinou** [comment]
Unfortunately, the Blender distribution is quite large, so having it as a hard dependency doesn't sound great to me.

I would prefer if we can call the Blender Flatpak externally somehow, so that it can be used as a soft dependency.

**Comment by lw64** [comment]
It would be of course possible to write a blender plugin that provides such an API, but that seems like work and also requires again setup effort. Another option is to add an extension point to the godot flatpak, where extensions can be mounted. Then with ideally one click you could install the blender extension and it all works.

**Comment by lw64** [comment]
Yeah, that would work too. I guess that it would be marked as "unsafe" doesn't matter too much.

**Comment by aliasbody** [comment]
Don't know if this is still open, but I was thinking about a bash script called "blender" that would search for either the OS-installed blender OR flatpak'd and server as a proxy for godot. In this case if the user has any compatible blender installed it would work, otherwise it would simply display the message it already displays.

I'll try to make an example and provide a pull-request to be evaluated.
